tests/testthat/test-carehomes-support.R

```diff
@@ -626,16 +657,22 @@ test_that("carehomes_compare combines likelihood correctly", {
   nms_sero_2 <- c("sero_pos_15_64_2", "sero_tot_15_64_2")
   nms_pillar2 <- c("pillar2_pos", "pillar2_tot")
   nms_pillar2_over25 <- c("pillar2_over25_pos", "pillar2_over25_tot")
+  nms_pillar2_age <- c("pillar2_under15_pos", "pillar2_under15_tot",
+                       "pillar2_15_24_pos", "pillar2_15_24_tot",
+                       "pillar2_25_49_pos", "pillar2_25_49_tot",
+                       "pillar2_50_64_pos", "pillar2_50_64_tot",
+                       "pillar2_65_79_pos", "pillar2_65_79_tot",
+                       "pillar2_80_plus_pos", "pillar2_80_plus_tot")
```

Comment:
split these into `nms_pillar2_under15` etc

Reply:
I've split them as nms_pillar2_age_under25 and nms_pillar2_age_over25. Is that ok?

Reply:
It needs to be mns_pillar2_under15, nms_pillar2_15_24, nms_pillar2_25_49 etc. We are testing every single data stream contributes to the likelihood - some data streams are a pair (pos/tot), and we need to list each pair individually